java-frontend/src/main/java/org/sonar/java/se/SymbolicExecutionVisitor.java

```diff
@@ -106,6 +115,32 @@ private boolean isObjectsRequireNonNullMethod(Symbol symbol) {
       return symbol.owner().type().is("java.util.Objects") && "requireNonNull".equals(symbol.name());
     }
 
+    @CheckForNull
+    private MethodBehavior createStringUtilMethodBehavior(Symbol.MethodSymbol symbol) {
+      MethodBehavior behavior;
+      switch (symbol.name()) {
+        case "isEmpty" :
+        case "isBlank" :
+        case "isNotEmpty" :
+        case "isNotBlank" :
+          behavior = new MethodBehavior(symbol);
+          MethodYield nullYield = new MethodYield(symbol.parameterTypes().size(), false);
+          nullYield.exception = false;
+          nullYield.parametersConstraints[0] = ObjectConstraint.nullConstraint();
+          nullYield.resultConstraint = symbol.name().contains("Not") ? BooleanConstraint.FALSE : BooleanConstraint.TRUE;
```

Comment:
I find having a condition on case values inside a switch statement confusing. Would be better to actually extract the method with parameter and call it with argument in right case branch